manifests/repo.pp

```diff
@@ -0,0 +1,26 @@
+class squid::repo
+{
+  if $::squid::manage_repo {
+    case $::osfamily {
+      'RedHat': {
+        case $::operatingsystemmajrelease {
+          '6','7': {
+            yumrepo { 'squid':
+              name     => 'Squid',
+              descr    => 'Squid repo for CentOS Linux - $basearch',
+              baseurl  => 'http://www1.ngtech.co.il/repo/centos/$releasever/$basearch/',
```

Comment:
baseurl should probably be a configuration variable also defaulting to this.

Generally why this repo? It seems a bit  random to pick this one to me? Or is this some how connected to
upstream?